linux/create_installer_linux.sh

```diff
@@ -124,6 +124,66 @@ for JDK_TARBALL in ${JDK_TARBALLS[*]} ; do
             ;;
     esac
 
+    # Checks the format of the VERSION variable. A consistent formatting is
```

Comment:
@johnoliver and @gdams - we have a shared lib for version parsing now don't we?  Please tell me we do :-D

Reply:
I just found https://github.com/AdoptOpenJDK/openjdk-build/blob/master/pipelines/library/src/common/VersionInfo.groovy which doesn't do what's needed here. I'm mainly interested in checking that the combination of the version and release type is legit.

Reply:
We do have a version parsing library (what @aahlenst linked to), but it probably would need to be enhanced in order to deal with some of this uses case, i.e add some sort of validate function, and probably parsing of the openj9 portions of the version string. But that is probably something we should do

Reply:
may be leaning towards accepting this unless we have time to upgrade the groovy lib some time soon

Reply:
OK apologies for the delay